.github/workflows/meson.yml

```diff
@@ -75,13 +77,25 @@ jobs:
         if: matrix.config.os == 'windows-latest' && matrix.config.msvc == true
         uses: ilammy/msvc-dev-cmd@v1
 
+      - name: Get time
+        if: matrix.config.msvc
+        id: get_time
+        run: |
+          "timestamp=$((Get-Date).Ticks)" >> $env:GITHUB_OUTPUT
+
+      - name: Restore cache
+        if: matrix.config.msvc
+        uses: actions/cache/restore@v4
+        with:
+          path: ${{ env.SCCACHE_DIR }}
+          key: windows-msvc-${{ steps.get_time.outputs.timestamp }}
+          restore-keys: windows-msvc-
+
       - name: Install MSVC dependecies with choco
         if: matrix.config.msvc
         run: |
-          # for whatever reason choco insist on a reboot if it installs meson
-          choco install ninja nasm python pkgconfiglite
-          python -m pip install --upgrade pip
-          pip install meson
+          python -m pip install meson
+          choco install sccache nasm
```

Comment:
The current comment is not true. I've added updated one. It's simply easier to manage PATH, when installed through pip.

Reply:
It definitely used to be true, but good to know it no longer does that.  
Can you elaborate on what issues there are with `PATH`? `nasm` is also installed via `choco` and seemingly shows up in the `PATH` without further action required.

Reply:
meson can find nasm in standard location, so it does not have to be in PATH. On the other hand, we call meson directly, so we have to have it in PATH. And since this jobs steps are shared, we would need to add some logic to add to path conditionally or call meson with absolute path. pip just seems clear to me

Also since meson is purely Python project, I find use of pip appropriate for the job, instead of some installers through.